src/main/java/co/willsalz/ttyl/application/TTYLApplication.java

```diff
@@ -111,9 +126,11 @@ public void run(final TTYLConfiguration config, final Environment env) throws Ex
                 phoneNumbers
         );
 
+
         // Register Resources
         env.jersey().register(new StartCallResource(callService));
         env.jersey().register(new ConnectCallResource());
+        env.jersey().register(new RepresentativeResource(representativeLookupGateway));
```

Comment:
ideally later we can move this to a service instead of passing the gateway around, especially if we want to back the representative lookup by a persistent store so we don't e.g. take down whoismyrepresentative.com :)